test

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+if [ -z $VIRTUAL_ENV ]; then
```

Comment:
Done. Added a comment and put a link to the general documentation, but that is a lot to read to understand anything.
source config.sh is somewhat similar to npm install.